Contents/Sketch/colors.js

```diff
@@ -7,7 +7,5 @@ var applyColors = function (newColors, docData) {
     color.alpha = newColors[i].Opacity;
     colors.push(color);
   }
-  colors = MSArray.dataArrayWithArray(colors);
-  appController.globalAssets().setPrimitiveColors(colors);
-  appController.globalAssets().objectDidChange();
```

Comment:
Thanks so much for working on this! `objectDidChange` is necessary for the inspector pane to update if a text layer is already selected. Is there a reason for removing this?